content/security/for-admins/enforce-sign-in/methods.md

```diff
@@ -1,48 +1,94 @@
-- When a user signs out, the **Sign in required!** prompt appears and they can
-  no longer use Docker Desktop.
+In some cases, a system reboot may be necessary for enforcement to take effect.
 
-> **Enforce sign-in vs enforce SSO**
->
-> Enforcing sign-in ensures that users are required to sign in to use Docker Desktop.
-> If your organization is also using single sign-on (SSO), you can optionally enforce SSO.
-> This means that your users must use SSO to sign in, instead of a username and password.
-> When you enforce sign-in and enforce SSO, your users must sign in and must use SSO to do so.
-> See [Enforce SSO](/security/for-admins/single-sign-on/connect#optional-enforce-sso) for details on how to enable this for your SSO connection.
-{ .tip }
+### Example deployment via Group Policy
 
+The following is only an illustrative example. 
 
-## Create a registry.json file to enforce sign-in
+There are many ways to deploy the registry key, for example using an MDM solution or with PowerShell scripting. The method you choose is dependent on your organizations infrastructure, security policies, and the administrative rights of the end-users. 
+
+1. Create the registry script. Write a script to create the `HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Docker\Docker Desktop` key, add the `allowedOrgs` string, and then set the value to your organization’s name.
+2. Within Group Policy, create or edit a Group Policy Objective (GPO) that applies to the machines or users you wish to target.
+3. Within the GPO, navigate to **Computer Configuration** > **Preferences** > **Windows Settings** > **Registry**.
+4. Add the registry item. Right-click on the **Registry** node, select **New** > **Registry Item**.
+5. Configure the new registry item to match the registry script you created, specifying the action as **Update**. Make suer you input the correct path, value name (`allowedOrgs`), and value data (your organization’s name).
+6. Link the GPO to an Organizational Unit (OU) that contains the machines you want to apply this setting to.
+7. Test the GPO. Test the GPO on a small set of machines first to ensure it behaves as expected. You can use the `gpupdate /force` command on a test machine to manually refresh its group policy settings and check the registry to confirm the changes.
+8. Once verified, you can proceed with broader deployment. Monitor the deployment to ensure the settings are applied correctly across the organization's computers.
+
+## plist method (macOS)
+
+1. Create the file `/Library/Application Support/com.docker.docker/desktop.plist`
+2. Open `desktop.plist` in a text editor and add the following content, where `myorg` is replaced with your organization’s name all lowercase:
+
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+   <plist version="1.0">
+     <dict>
+	     <key>allowedOrgs</key>
+	     <string>myorg</string>
```

Comment:
```suggestion
         <array>
	         <string>myorg</string>
	     </array>
```

⚠️ needs to be changed. Apologise for this, I tweaked the log-in enforcement guide on Friday due to some required last-minute changes.